src/pages/Room/index.js

```diff
@@ -168,10 +156,7 @@ const Room = () => {
             {loading ? <Loading /> : null}
             <ErrorMessage>{error ? error : null}</ErrorMessage>
```

Comment:
What's the reason why you need to throw `null` if it doesn't have any error

so what if we're doing like if error ? render then ErrorMessae : render nothing

Reply:
@flukebevil Hi brother, I will extract 2 answers.

1. Return `null` to render is fine it's mean nothing to render.
2. Passed `null` to children I'm not sure because the ErrorMessage is rendered but without the value of children is shown or not.

Reply:
@KanRMUTL i recommend 

```js
{error && <ErrorMessage>{error}</ErrorMessage>} // solution 1
{error ? <ErrorMessage>{error}</ErrorMessage> : null} // solution 2
```

Reply:
I choose solution 1 because we don't have a reason for rendering `null`

Reply:
@T-Jedsada Got it.

@KanRMUTL it fine bro haha

i just ask for my knowledge is it possible to check something to decide to render this component or not. haha

Reply:
@KanRMUTL it's ok to choose solution 1 but solution 2 is also made a bit clearer.

solution will have some problem (but not in this case) in that if error is `0` our UI will show 0 for this case

![Screen Shot 2564-11-02 at 18 54 09](https://user-images.githubusercontent.com/13183413/139841477-26a50648-2594-4ab6-ae2c-55f28c8d8d62.png)
.png)


So there's still reason to use solution 2